adoc/admin-dr-backup-etcd.adoc

```diff
@@ -0,0 +1,180 @@
+= Backup etcd Cluster Data
+This chapter describes the backup of etcd cluster data running on master nodes of {productname}.
+
+== Data To Backup
+1. Create backup directories on external storage.
++
+```bash
+BACKUP_DIR=CaaSP_Backup_`date +%Y%m%d%H%M%S`
+mkdir /${BACKUP_DIR}
+```
+
+2. Copy the following files/folders into the backup directory:
+* The `skuba`: command-line binary for the running cluster, use to replace nodes from cluster.
+* The skuba cluster folder: directory where used for running `skuba` commands. Contains cluster configurations and authentication files.
+* The etcd cluster database: holds all cluster data. Can be used to recover master nodes. Please refer to the next section for steps to create etcd cluster database backup.
+
+3. (Optional) Make backup directory into a compressed file, and remove the original backup directory.
++
+```bash
+tar cfv ${BACKUP_DIR}.tgz /${BACKUP_DIR}
+rm -rf /${BACKUP_DIR}
+```
```

Comment:
Should be deleted?

Reply:
IMO this block is relevent unless we assume they're skuba is in-version and "my-cluster" folder has all the files..?